### PR TITLE
some light editing to get this page to standard

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Noticed that this page needed some love when I was going through looking at bad links. Haven't done a lot, just enough to stop the linter from having conniptions.

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: timescaledb/timescaledb-edition-comparison.md [hunk 1]
```diff
@@ -5,39 +5,65 @@ keywords: [Apache, community, features]
...
+TimescaleDB Community Edition is the latest, most updated version of TimescaleDB,
+available under the
+[Timescale License (TSL)][timescale-license].
+
+For more information about the Timecale licence, see [this blog post][license-blog].
+
+Many of the most recent features of TimescaleDB are only available in
+TimescaleDB Community Edition.
+
+You can install TimescaleDB Community Edition in my own on-premises or cloud
```
Comment (charislam) [inline timescaledb/timescaledb-edition-comparison.md:48]:
```suggestion
You can install TimescaleDB Community Edition in your own on-premises or cloud
```

File: timescaledb/timescaledb-edition-comparison.md [hunk 2]
```diff
@@ -5,39 +5,65 @@ keywords: [Apache, community, features]
...
+
+You cannot sell TimescaleDB Community Edition as a service, even if you are the
+main contributor.
 
-Many of the most recent features of TimescaleDB are only available in TimescaleDB Community Edition.
+You can modify the TimescaleDB Community Edition source code and run it for
+production use. Developers using TimescaleDB Community Edition have the "right
+to repair" and make modifications to the source code and run it in their own
+on-premises or cloud infrastructure. However, consistent with the previous
+question, users may not make modifications to the TimescaleDB Community Edition
```
Comment (charislam) [inline timescaledb/timescaledb-edition-comparison.md:59]: There aren't any questions anymore so this is a bit confusing ;)

**Review by charislam — APPROVED** [review]

File: timescaledb/timescaledb-edition-comparison.md [hunk 3]
```diff
@@ -5,39 +5,65 @@ keywords: [Apache, community, features]
...
+*   [Aiven for PostgreSQL][aiven]
 
 ### TimescaleDB Community Edition
 
-TimescaleDB Community Edition is the latest, most updated version of TimescaleDB, available under the [Timescale License (TSL)][timescale-license]. [Read more about the Timescale License][license-blog].
+TimescaleDB Community Edition is the latest, most updated version of TimescaleDB,
+available under the
+[Timescale License (TSL)][timescale-license].
+
+For more information about the Timecale licence, see [this blog post][license-blog].
```
Comment (Loquacity) [inline timescaledb/timescaledb-edition-comparison.md:43]:
```suggestion
For more information about the Timescale licence, see [this blog post][license-blog].
```

File: timescaledb/timescaledb-edition-comparison.md (hunk 2 above)
Comment (Loquacity) [inline timescaledb/timescaledb-edition-comparison.md:59]:
```suggestion
on-premises or cloud infrastructure. However, you cannot make 
modifications to the TimescaleDB Community Edition
```

Comment (Loquacity) [reply timescaledb/timescaledb-edition-comparison.md:59]: oof, missed that bit 🤦‍♀️

**Pull request merged**